multi_wikiscrape.py

```diff
@@ -0,0 +1,141 @@
+import scrapy
+import sys
+import logging
+from datetime import datetime
+from twisted.internet import reactor
+from scrapy.crawler import CrawlerRunner
+from scrapy.utils.log import configure_logging
+from scrapy.utils.project import get_project_settings
+
+log = logging.getLogger(__name__)
+
+#mongoDB connect DONE in settings/pipeline
+
+class WikiSpider1(scrapy.Spider):
```

Comment:
There are 3 spiders in this project that have the same goal, we should only have one spider (even if we run it multiple times) unless they scrape different kinds of data (scraping image files instead of articles, ect.).